Tools/CISupport/ews-build/master.cfg

```diff
@@ -52,7 +52,7 @@ c['www']['change_hook_dialects'] = dict(
             'github.head.ref',
             'github.base.sha',
             'github.head.repo.full_name',
-        ],
+        ], 'token': load_password('GITHUB_COM_ACCESS_TOKEN'),
```

Comment:
Do we need to configure username as well, or just the token?

Reply:
According to the buildbot documentation, just the token. That's not entirely surprising, they stick the token in a header. That's not how we do authentication in our code (we use basic auth as the GitHub API advertises), but it seems like that's how GitHub's tokens work.